packages/vats/src/core/chain-behaviors.js

```diff
@@ -97,6 +97,21 @@ export const bridgeCoreEval = async allPowers => {
 };
 harden(bridgeCoreEval);
 
+/**
+ * @param {BootstrapPowers} powers
+ */
+export const makeOffChainBrands = async ({
+  brand: {
+    produce: { USD: usdBrandProducer },
+  },
+}) => {
+  // Create the USD brand referred to by price oracles.
```

Comment:
I suggest moving this comment to the function JSDoc comment. It's not just an implementation detail.